### PR TITLE
Add ZTF alerts notebook

### PR DESCRIPTION
**Pull request by hombit**
Add pre-executed notebook with "Alerce" data.

Rendered doc: https://lsdb--479.org.readthedocs.build/en/479/tutorials/pre_executed/ztf-alerts-sne.html

### CODE REVIEW

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 1]
```diff
@@ -0,0 +1,2117 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fc1a0e0de461de1",
+   "metadata": {},
+   "source": [
+    "# Search for SN-like light curves in ZTF alerts\n",
+    "\n",
+    "We will use lsdb package to load a Hats catalog with [ZTF](https://www.ztf.caltech.edu) alerts.\n",
+    "The dataset contains all alerts sent from the beginning of the survey until 2023-09-13 corresponding to objects having at least 20 detections.\n",
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:11]: Do you know when was the beginning of the survey?

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 2]
```diff
@@ -0,0 +1,2117 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fc1a0e0de461de1",
+   "metadata": {},
+   "source": [
+    "# Search for SN-like light curves in ZTF alerts\n",
+    "\n",
+    "We will use lsdb package to load a Hats catalog with [ZTF](https://www.ztf.caltech.edu) alerts.\n",
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:10]: Hats -> HATS

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 3]
```diff
@@ -0,0 +1,2117 @@
...
+   "execution_count": 2
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccfb5d53-6229-400c-8dfc-b2972de158ec",
+   "metadata": {},
+   "source": [
+    "### Helper function for light-curve plotting\n",
+    "\n",
+    "The function accepts a pandas data frame and plot a light curve."
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:78]: plot -> plots

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 4]
```diff
@@ -0,0 +1,2117 @@
...
+   "id": "dde7cda2-4e05-493f-9940-75424c3e31b8",
+   "metadata": {},
+   "source": [
+    "### Load catalog structure and \"nest\" columns\n",
+    "\n",
+    "Here we load the catalog structure and transform list-columns to a compact representation.\n",
+    "These transformed columns are \"nested data frames\", so each item could be represented by a small pandas dataframe.\n",
+    "We are going to have three nested columns:\n",
+    "\n",
+    "1. \"lc\", for light curves, each point corrersponds to some alert (detection)\n",
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:121]: corresponds -> corresponding

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 5]
```diff
@@ -0,0 +1,2117 @@
...
+    "2. \"nondet\", for non-detections (upper limits)\n",
+    "3. \"ref\", for ZTF reference objects associated with alerts\n",
+    "\n",
+    "Here we do not download any data yet, all data access and analysis happens only after `.compute()` is called.\n",
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:125]: Here we have not downloaded ....

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 6]
```diff
@@ -0,0 +1,2117 @@
...
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3458764513820540928</th>\n",
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:701]: It is super confusing that these two catalogs have the same name

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb [hunk 7]
```diff
@@ -0,0 +1,2117 @@
...
...SIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIIECJPDnn3/KgAED5IADDpBzzjmnTBLYsGGD9OzZUy699FJp0qRJia/x66+/NswGDx4sO+ywQ4nrYwUkUFYJULEoq3eW10UCIQTOPfdcefjhh+XZZ5+V5s2bh0jm76G//vpLjj/+eFm8eLHMmzdPDjvssBJdzO233y69evWSzz//XHbdddcS1cXCJFCWCVCxKMt3l9dGAkkIDB8+XK688krp16+fDBw4MIlE2cn68ssvpX79+uaCoGDstddexb64Aw88UHbeeWd56aWXil0HC5JAIRCgYlEId5nXSAL/J/D8889Lq1at5MgjjzQd5BZbbBHI5qeffpIPP/xQGjRoECiTqQNxnvvFF1+UFi1aSJ06deTNN9+UrbbaKu1mL126VBo2bCj333+/XHjhhWmXZwESKCQCVCwK6W7zWguawD///CP77LOPrFmzRl555RU56qijAnmsWLHCTJFg2P+9994znXKgcMwHMnHu1q1by9NPPy2jRo2SLl26pN3iK664QkaPHi0YAaF9Rdr4WKDACFCxKLAbzsstXAIzZ86
```
Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:1023]: These images render very big for me, much bigger than they should be

Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:1288]: The code will download ...

Comment (nevencaplar) [inline docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:1289]: we dont have any way to monitor the downloading progress, do we?

**Review by nevencaplar — APPROVED** [review]

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb (hunk 1 above)
Comment (hombit) [reply docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:11]: I found the minimal MJD with LSDB =)

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb (hunk 7 above)
Comment (hombit) [reply docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:1289]: I added a note about dask dashboard

File: docs/tutorials/pre_executed/ztf-alerts-sne.ipynb (hunk 6 above)
Comment (hombit) [reply docs/tutorials/pre_executed/ztf-alerts-sne.ipynb:670]: Yes, but it is out of scope...

**Pull request merged**